specification/ml/infer_trained_model/MlInferTrainedModelRequest.ts

```diff
@@ -45,13 +45,14 @@ export interface Request extends RequestBase {
   }
   body: {
     /**
-     * An array of objects to pass to the model for inference. The objects should contain a field matching your
-     * configured trained model input. Typically, the field name is `text_field`. Currently, only a single value is allowed.
+     * An array of objects to pass to the model for inference. The objects should contain a fields matching your
+     * configured trained model input. Typically, for NLP models, the field name is `text_field`.
+     * Currently, for NLP models, only a single value is allowed.
      */
-    docs: Dictionary<string, string>[]
+    docs: Dictionary<string, any>[]
```

Comment:
Instead of `any` we should use `UserDefinedValue` for our "any" type as it has special meaning in implementing languages.

Reply:
ok, as long as it can capture pretty much any value :)